item_values.go

```diff
@@ -5,6 +5,8 @@ import (
 	"net/http"
 )
 
+// AddItemValues sets/updates (some) property values of the given item. The properties (columns) must be previously
```

Comment:
(some) is misleading. Either remove or change description

Reply:
It's copied verbatim from the official documentation: https://docs.recombee.com/api.html#set-item-values

Reply:
I dont approve event official documentation, remove it.